lib/rspec/html_messages.rb

```diff
@@ -81,16 +81,28 @@ def backtrace_html(**options)
       render_template("_backtrace")
     end
 
+    def status_html(**options)
+      css_class, message = ""
+      if passed?
+        css_class = "alert-success"
+        message = "This test passed!"
+      else
+        css_class = "alert-danger"
```

Comment:
```suggestion
        css_class = "alert-warning"
```

I was thinking of yellow for failures (and avoiding the word "fail" in messages), to be less scary. I used `warning` below too for the failure message card bg-color.